contracts/test/ExampleRollup.t.sol

```diff
@@ -1,64 +0,0 @@
-// SPDX-License-Identifier: UNLICENSED
```

Comment:
why did we delete this test and not replace with a new test?

Reply:
True, I wasnt focusing a lot on testing because it was an example but I will add something

Reply:
testing is not dire to be fair - as it's just an example but some things I would test would be testing that the errors you added are triggered in the scenarios that you expect them to be triggered. But def not necessary

Reply:
okay makes sense! I will give the tests a go and see where I land

Reply:
https://github.com/EspressoSystems/sequencer-example-l2/commit/9c85050f35e37c3dde45993b36089da11edafab4